### ISSUE TITLE
Track inpatient unit where medications where administered 

### ISSUE THREAD

**Issue by rwaitman** [CDMFORUM/CDM-GUIDANCE#16]
Piloting capture of inpatient unit would be a needed data element for antimicrobial use and reporting project with the FDA and CDC.  Bill Trick (CAPriCORN) Investigator.  Rick Platt, Executive Committee member facilitator.

**Comment by mgkahn** [comment]
Please attach the relevant projects to your post. It's difficult to understand the justification ("nice to have" versus "critical for analytics") for this request from your posting. Is it possible to provide more details about how this data element would be used in these projects? Would help thinking about how best to capture/represent.

**Comment by rwaitman** [comment]
![inpatientantimicrobialproject](https://cloud.githubusercontent.com/assets/15934088/16126034/fd51e9ea-33bb-11e6-8f7e-4cc454ad1cca.jpg)

Photo of Rich's project description attached

**Comment by dckc** [comment]
@russwaitman, is this the same issue as clinical service in TAPIR and the National Diabetes Amputation Prevention Study?

ref
 - [Possibility of adding clinical service to GPC ontology](http://listserv.kumc.edu/pipermail/gpc-dev/2016q2/002953.html) Mei Liu Wed Jun 15 09:54:09 CDT 2016

**Comment by rwaitman** [comment]
No.  That's related to issue 17.  As the figure says, this issue is the physical location of the hospital unit where the patient was located with antibiotics were used.

**Comment by mgkahn** [comment]
Not sure how much discussion you want to occur here or if this is better handled by the Data Model subgroup. I think what is being requested is replicating location mapping as described in this presentation on the CDC NHSN web site from 2012: https://www.signup4.net/Upload/AVAR10A/TEST2089E/2-%20Location%20Mapping%20slideset.pdf

If I understand NHSN data collection (which I admit that I don't), locations are used to describe patients with infections, not with antibiotic administration events. To move down to the level of location with every antibiotic administration event would  require access to the eMAR so you know the exact time every antibiotic dose was given. You also need access to detailed ADT information ("ADT Events" in Epic-eeze) that gives the date-times that a patient is admitted into a unit and discharged from that unit. You then need to do the temporal interval joins across all of these event (big tables!) to brings these time stamps into alignment to know the patient's unit at the time of antibiotic administration. 

Even if the only information desired is the various units that the patient occupied during their inpatient stay, you would need access to the detailed ADT events for all the admit/discharge into/out of units events. In our institution, this table is extremely large and has a rather significant daily growth rate unless it could be narrowed to specific subpopulations.

Do I have this right? Hopefully there is something much simpler in the minds of the investigators/requesters.

**Comment by dckc** [comment]
Where do I look for discussions of the Data Model subgroup?

**Comment by rusincovitch** [comment]
Hi @dckc: I'd defer to the workgroup on their planning. The workgroup's [central wiki page ](https://github.com/PCORnet/DataCommittee/wiki/Develop-Annual-Updates-of-the-Common-Data-Model) provides a good starting point?

**Comment by marey5** [comment]
To help add complexity and confusion, we also refer to our nursing units by their physical location, but these may change over time as the hospital moves certain patient populations to different floors.  When the nursing unit moves (along with their population), their label is updated accordingly.  To understate things, this makes longitudinal analyses tough.  

So if the study were only looking at the physical characteristics of the space, then a simple label is fine.  But if there were also sub-aims looking at practice patterns or anything time-based, physical unit is not going to be sufficient.

**Comment by smerek** [comment]
Once CDM v4.0 is implemented, potential options would be to extend the MED_ADMIN table or to use the OBS_GEN table to store this information.
